momentumopt/python/momentumopt/robots/blmc_robot_wrapper.py

```diff
@@ -256,6 +257,54 @@ def set_init_config(self):
         self.set_configuration(self.q)
 
 
+class BipedWrapper(BasicRobotWrapper):
+
+    def __init__(self, q=None):
+        super(BipedWrapper, self).__init__()
+
+        self.effs = ["FL", "FR"]
+        # self.colors = {"L": "r", "R": "y"}
+        self.joints_list = ["HAA", "HFE", "KFE", "ANKLE"]
+        self.floor_height = 0.
+
+        self.robot = BoltConfig.buildRobotWrapper()
+
+        self.num_ctrl_joints = 6
+
+        # Create data again after setting frames
+        self.model = self.robot.model
+        self.data = self.robot.data
+        q = pinocchio.neutral(self.robot.model)
+        if not q is None:
+            self.set_configuration(q)
+        else:
+            self.q = None
+        self.M_com = None
+        self.mass = sum([i.mass for i in self.model.inertias[1:]])
+        self.set_init_config()
+
+    def set_init_config(self):
+        model = self.model
+        data = self.data
+        NQ = model.nq
+        NV = model.nv
+        self.q, self.dq, self.ddq, tau = zero(NQ), zero(NV), zero(NV), zero(NV)
+
+        self.q = pinocchio.neutral(self.robot.model)
+        self.q[2] = self.floor_height
+
+        # Set initial configuration
+        angle = np.deg2rad(60.0)
+        q_dummy = np.zeros(self.num_ctrl_joints)
+        q_dummy[2::3] = angle
+        q_dummy[1:3] = -0.5 * angle
+
+        self.q[7:] = q_dummy
+
+        # print(self.q)
+        self.set_configuration(self.q)
+
+
 ############################ For debugging ##########################################
 
-# robot = QuadrupedWrapper()
+# robot = BipedWrapper()
```

Comment:
If this is a comment here, should we remove it?

Reply:
sure, any more comments?

Reply:
no, you are good to go